contracts/package.json

```
@@ -73,7 +73,8 @@
     "prettier:list-different": "prettier --list-different \"**/*.{js,json,md,sol,ts}\"",
     "test": "concurrently --success first --kill-others \"hardhat node &>/dev/null\" \"hardhat test --network localhost\"",
     "typechain": "hardhat typechain",
-    "prepare": "yarn run typechain"
+    "prepare": "yarn run typechain",
+    "deploy:umbra": "hardhat run scripts/deploy.js"
```

Nit, but maybe rename the deploy script? Since there's only one we don't need the `:umbra` suffix. Also good to explicitly compile beforehand
```suggestion
    "deploy": "yarn build && hardhat run scripts/deploy.js"
```

---

Good idea.. done